src/app_service/service/chat/dto/chat.nim

```diff
@@ -10,8 +10,7 @@ type ChatType* {.pure.}= enum
   Public = 2,
   PrivateGroupChat = 3,
   Profile = 4,
-  Timeline = 5
-  CommunityChat = 6
+  CommunityChat = 5
```

Comment:
I believe this change is incorrect. The chat type is an enum specifying the type of the chat. Just because we're removing the timeline, doesn't mean that chatType == 5 all of a sudden means "community chat".

Reply:
done

> I think more QML can be removed I think the way we display message is different for timeline and as such can be removed too. I believed this is this:
> https://github.com/status-im/status-desktop/blob/master/ui/imports/shared/views/chat/MessageView.qml#L330-L356
> (but please double check ;))

Checking that I see that `chatType: 4` is  `Profile` :
```
type ChatType* {.pure.}= enum
  Unknown = 0,
  OneToOne = 1, 
  Public = 2,
  PrivateGroupChat = 3,
  Profile = 4,
  
```
could somebody confirm that `Profile` corresponds to readonly `property int chatTypeStatusUpdate: 4` from Contants.qml? @PascalPrecht git blames you for the implementation, commit `f512d5063c5853dcdee9670e17defbcde4275fbe` "feat: introduce StatusUpdateChatInput variation"

Reply:
done